samples/algorithms/GHZ.qs

```diff
@@ -18,6 +18,7 @@
 namespace Sample {
     open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Arrays;
```

Comment:
I think this was left behind from an earlier draft, it doesn't seem to be used now.

Reply:
Removed unnecessary open statements here and in Cat sample. Do we have lints yet?